runtime-config.md

```diff
@@ -1,3 +1,14 @@
+# Runtime Configuration
+
+## Hooks
+
+Lifecycle of a container's runtime, for setup and teardown.
```

Comment:
This doesn't parse for me.  In #126 I moved the hooks section from the current `runtime.md` (which is about the lifecycle) over to this section (which is about `runtime.json` and configuring hooks).